draft-ietf-rtcweb-jsep.xml

```diff
@@ -1642,7 +1642,10 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
               <t>Each "m=" and c=" line MUST be filled in with the port, protocol,
               and address of the default candidate for the m= section, as described
-              in <xref target="RFC5245"></xref>, Section 4.3. Each
+              in <xref target="RFC5245"></xref>, Section 4.3. If ICE checking
+              has already completed for one or more candidate pairs and a candidate
+              pair is in active use, then that pair MUST be used, even if ICE
+              has not yet completed. Each
```

Comment:
Do you want to add a reference to 5245, S 9.1.2.2 here, or mention how this slightly differs from the guidance offered there (which refers to ICE completed)?
https://tools.ietf.org/html/rfc5245#section-9.1.2.2